src/auth/oauth2client.ts

```diff
@@ -532,37 +537,43 @@ export class OAuth2Client extends AuthClient {
 
   /**
    * Verify id token is token by checking the certs and audience
-   * @param {string} idToken ID Token.
-   * @param {(string|Array.<string>)} audience The audience to verify against the ID Token
+   * @param {VerifyIdTokenOptions} Object that contains all options.
    * @param {function=} callback Callback supplying GoogleLogin if successful
    */
-  verifyIdToken(idToken: string, audience: string|string[]):
-      Promise<LoginTicket|null>;
+  verifyIdToken(options: VerifyIdTokenOptions): Promise<LoginTicket|null>;
   verifyIdToken(
-      idToken: string, audience: string|string[],
+      options: VerifyIdTokenOptions,
       callback: (err: Error|null, login?: LoginTicket|null) => void): void;
   verifyIdToken(
-      idToken: string, audience: string|string[],
+      options: VerifyIdTokenOptions,
       callback?: (err: Error|null, login?: LoginTicket|null) => void):
       void|Promise<LoginTicket|null> {
+    // This funtion used to accept two arguments instead of an options object.
+    // Check the types to help users upgrade with less pain.
```

Comment:
suggestion: add a comment that this check can be removed after a 2.0 release.

Reply:
fixed!